app/services/mlh_table.rb

```diff
@@ -28,11 +28,8 @@ def faraday_connection
         )
       end
     end
-    if response.success?
-      response.body
-    else
-      AirtablePlaceholderService.call('Meetups')
-    end
+    response.body if response.success?
+    { 'data' => AirtablePlaceholderService.call('Meetups') }
```

Comment:
```suggestion
    return response.body if response.success?
    { 'data' => AirtablePlaceholderService.call('Meetups') }
```
We're not returning the valid response if we have it, just missing `return`